### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

**Comment by roj1512** [comment]
Which one is correct?

- grammY api reference
- grammY API reference
- grammY API Reference

**Comment by KnorpelSenf** [comment]
> Which one is correct?
> 
> * grammY API reference
> * grammY API Reference

I have no idea.

**Review by KnorpelSenf — APPROVED** [review]
I was waiting for this lol, would've done it myself in a few days

**Comment by KnorpelSenf** [comment]
grammY api reference *hides away*

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]
Let's use _grammY API reference_ as we do [here](https://grammy.dev/guide).

File: README.md [hunk 1]
```diff
@@ -96,19 +96,19 @@ Includes a setup to easily run any of them.
...
 
-### [grammY News Channel](https://t.me/grammyjs_news)
+### [grammY news](https://t.me/grammyjs_news)
 
-—Telegram channel where updates to grammY and the ecosystem are posted.
+—The channel where updates to grammY and the ecosystem are posted.
 We are also [on Twitter](https://twitter.com/grammy_js).
 
-### [Telegram Bot API Reference](https://core.telegram.org/bots/api)
+### [Telegram Bot API reference](https://core.telegram.org/bots/api)
```
Comment (KnorpelSenf) [inline README.md:111]: Now you're using grammY style for TG stuff. Shouldn't it be title case? Yes, we will then have two different casings of the same word

**Comment by roj1512** [comment]
I modified the wrong line.

**Review by KnorpelSenf — APPROVED** [review]
Thanks!

**Pull request merged**